src/shared/containers/join-us/index.js

```diff
@@ -88,17 +81,11 @@ const apply = {
   }
 };
 
-const listing = {
-  title: 'Badger Academy Internships',
-  body: '<p>Badger Academy is a little bit different to most internship programmes.</p><p>At the beginning of each summer we take on a group of students; usually around 6. They spend 2 months with us immersing themselves in life at Red Badger and learning about what it’s really like to work in a professional software workshop.</p>',
-  href: '/badger-academy-internships'
-};
-
-const listings = (
```

Comment:
I am wondering if this is overkill? Should `join-us` be a container component that connects, then we just pass the jobs prop into jobs? Maybe a definition of container component would help.